nipype/utils/draw_gantt_chart.py

```diff
@@ -322,10 +322,7 @@ def draw_resource_bar(start_time, finish_time, time_series,
     space_between_minutes = space_between_minutes / scale
 
     # Iterate through time series
-    if PY3:
-        ts_items = time_series.items()
-    else:
-        ts_items = time_series.iteritems()
+    ts_items = list(time_series.items())
```

Comment:
This seems to be specifically undoing the iterable-ness of `ts_items`. Why?

Reply:
it runs `len` a couple of lines below. I can roll back this change if you want (tried to remove all the `iteritems` that were remaining).

Reply:
You were right on this one, the `len` is calculated on a different variable. Fixed.